pkg/platform/common/vmlayer/vmprovider.go

```diff
@@ -488,13 +489,14 @@ func (v *VMPlatform) InitHAConditional(ctx context.Context, platformConfig *plat
 			}
 		}()
 	} else {
+		rootLbAction = platform.ActionCreate
 		err := v.initRootLB(ctx, platformConfig, ActionCreate, updateCallback)
 		if err != nil {
-			return err
+			return platform.ActionNone, err
```

Comment:
Is there any reason we shouldn't just call CheckRebuildRootLb() right here after initRootLB() is called for the first time? I don't really see any need for it to be run from the CRM. Only if we manually deleted the rootLB will it incur any extra real delay, which will almost never be the case. Not all platforms may necessarily have rootLBs, either.

Reply:
We could yes(initially I thought caches wouldn't be ready there, but they got initialized way before the call to initHAConditional) - we'll just need to pass caches to InitHAConditional(), which I didn't want to do, but it makes sense - this way there is no need to make checkRebuildRootLb() part of the platform interface. Let me change that.

Reply:
Cool, thanks. And you don't need to pass caches to InitHAConditional, they're already stored on VMPlatform.Caches as part of InitCommon().